bin/deploy

```diff
@@ -4,11 +4,27 @@
 #
 #   APP defaults to carnival-staging
 #
+#   Set BUILD=1 to perform a Halcyon build
+#
 ###
 set -e
 
 case "${1:-carnival-staging}" in
-  carnival-staging) git push staging master ;;
+  carnival-staging)
+    git push --force staging master
+
+    if [ "$BUILD" = 1 ]; then
```

Comment:
No reason. I just usually err on the explicit and exact side. If you want OHYEAH though, you can have it.